src/components/GenericDropdown.tsx

```diff
@@ -24,6 +24,7 @@ const Container = styled.div`
 
 function GenericDropdown({ buttonDisplay, dropdownItems }: Props) {
 	const ref = useRef<HTMLDivElement>(null);
+	const buttonRef = useRef<HTMLButtonElement>(null);
 	const { isComponentVisible, setIsComponentVisible } = useComponentVisible(
 		ref,
 		true
```

Comment:
This is probably out of scope for this PR (especially since you didn't even make a change on this!!) but it feels like the default here should be false so that dropdowns are not visible when the page loads!

Reply:
Hmm that is a good point. @lbare can you make this change?